rpdrtools/io/io.py

```diff
@@ -0,0 +1,201 @@
+import csv
+import logging
+from pathlib import Path
+from typing import Generator, List, Union
+
+import pandas as pd
+from rich.progress import Progress
+from ..constants import (
+    RPDR_NEWLINE_CHAR,
+    RPDR_FILE_DELIMITER,
+    RPDR_REPORT_TEXT_FIELD,
+    RPDR_REPORT_END_TOKEN,
+)
+from .types import OnBrokenRecordsType
+
+
+def _merge_rows(
+    row1: List[str], row2: List[str], newline_char: str = RPDR_NEWLINE_CHAR
```

Comment:
I think the default newline should just be "\n". In general I think we'd want to use the "\r\n" newline character as little as possible. The input files have to have it, but I don't think anyone wants the "\r" character inside their dataframes or whatever.

Reply:
Decided to leave it as it is. Python's csv module is similar: it preserves Windows-style carriage returns inside quoted fields.